tachyon_core/src/execution/node/vector_select.rs

```diff
@@ -68,15 +68,19 @@ impl VectorSelectNode {
 
 impl ExecutorNode for VectorSelectNode {
     fn value_type(&self) -> ValueType {
-        self.cursor.value_type()
+        // will only default to UInteger64 if this node iterates over 0 files
+        // TODO: could potentially query indexer to get the value type in this case
+        self.cursor.value_type().unwrap_or(ValueType::UInteger64)
```

Comment:
Don't believe the value type matters if there are 0 values returned - lmk if this assumption is incorrect

Reply:
Yeah this probably doesn't matter, but if we default to a type I feel that `f64` is more common since the CLI defaults to that as well